src/det.jl

```diff
@@ -47,7 +47,7 @@ end
     if prod(S) ≤ 14*14
         quote
             @_inline_meta
-            LUp = lu(A)
+            LUp = lu(A; check = false)
```

Comment:
It may be slightly helpful to call `_lu` directly here. In other places in this package we've found that keyword arguments still sometimes cause suboptimal performance. (I'm not sure about this particular case, we never dug to the real root cause in the compiler yet.)

Reply:
Of course, I will change it.